lib/common/asar.js

```diff
@@ -252,9 +252,13 @@
     }
 
     const { lstat } = fs
-    fs.lstat = (pathArgument, callback) => {
+    fs.lstat = function (pathArgument, options, callback) {
       const { isAsar, asarPath, filePath } = splitPath(pathArgument)
-      if (!isAsar) return lstat(pathArgument, callback)
+      if (arguments.length < 3) {
```

Comment:
I think this would be more consistent / reliable

```suggestion
      if (typeof options === 'function') {
```

Reply:
interesting, i took my cue from node here